CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Design Systems Coalition NYC is to be inclusive to the largest number of members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
Not a comment on the actual code of conduct, more on event planning.
Since we're calling out "ability" (as we should) we should check that all event spaces are wheelchair accessible and also talk about what to do for any visually or hearing impaired guests want to attend or speak.